crates/viewer/re_renderer/src/video/player.rs

```diff
@@ -181,9 +182,8 @@ impl VideoPlayer {
         }
     }
 
-    fn frame_at_internal(
+    fn unqueue_samples(
```

Comment:
I think you meant this instead :)
```suggestion
    fn enqueue_samples(
```